### PR TITLE
DAC6-2738: Add SecondContactCanWePhonePage

### PR DESCRIPTION
**Pull request by koladeadewuyi**
To test this visit:
- /
- /nameOfFinancialInstitution
- /second-contact-name
- and then /second-contact-have-phone 

Screenshot 2024-01-17 at 13 51 46

### CODE REVIEW

File: app/views/SecondContactCanWePhoneView.scala.html [hunk 1]
```diff
@@ -32,15 +32,19 @@
             @govukErrorSummary(ErrorSummaryViewModel(form))
         }
 
+        <h1 class="govuk-heading-l">@messages("secondContactCanWePhone.heading", secondContactName)</h1>
```
Comment (C-Thomp) [inline app/views/SecondContactCanWePhoneView.scala.html:35]: Minor change but we can use the view component for headings and paragraphs 😎 - 
```suggestion
  @heading(messages("secondContactCanWePhone.heading", secondContactName))
```

File: app/controllers/IndexController.scala [hunk 2]
```diff
@@ -17,23 +17,36 @@
...
     with I18nSupport {
 
-  def onPageLoad: Action[AnyContent] = identify {
+  def onPageLoad: Action[AnyContent] = identify.async {
     implicit request =>
-      Ok(view())
+      sessionRepository.set(UserAnswers(request.userId)) map {
+        case true => Ok(view())
+        case false =>
+          logger.error(s"Failed to initialize user answers for userId: [${request.userId}]")
```
Comment (C-Thomp) [inline app/controllers/IndexController.scala:47]: is this so we dont use the index as a landing page before starting the journey?

Comment (koladeadewuyi) [reply app/controllers/IndexController.scala]: It's so the userAnswers object is initialised when the index page is visited. This is to ease testing by the testers until we have the full journey implemented.

**Review by C-Thomp — APPROVED** [review]
🥳

**Pull request merged**